lib/atat-net-stack.ts

```diff
@@ -1,19 +1,39 @@
-import * as ec2 from "@aws-cdk/aws-ec2";
-import { InterfaceVpcEndpointAwsService } from "@aws-cdk/aws-ec2";
-import * as cdk from "@aws-cdk/core";
-import * as custom from "@aws-cdk/custom-resources";
+import * as cdk from "aws-cdk-lib";
+import * as ec2 from "aws-cdk-lib/aws-ec2";
+import * as custom from "aws-cdk-lib/custom-resources";
+
+import { Construct } from "constructs";
+
+const DEFAULT_ROUTE = "0.0.0.0/0";
 
 export interface AtatNetStackProps extends cdk.StackProps {
+  /**
+   * The CIDR block to use for the VPC.
+   *
+   * If this is not provided, the default value from the CDK's VPC construct
+   * will be used. VPCs with overlapping ranges may cause routing issues for
+   * the application. This value should almost always
```

Comment:
Unfinished sentence here.  _Should almost always..._

Reply:
Good catch. Changed to:

>    * the application. This value should almost always be provided.